packages/react/src/components/ui/buttons/SecondaryButton.tsx

```diff
@@ -28,7 +28,10 @@ export const SecondaryButton: FC<PropsWithChildren<SecondaryButtonProps>> = ({
   }`;
 
   return (
-    <span onClick={handleClick}>
+    <button
```

Comment:
why do we need this change?

Reply:
yes, because of two reasons:
- logically it makes sense for a SecondaryButton to be of type button.
- It makes easier to point to this button in e2e test as we have try again text written 2 times on the ErrorScreen. 
We can add additional attributes to the Try again button to fix the second point but because of the first point it made more sense to make this change.